install/kubernetes/quick-install.yaml

```diff
@@ -42,6 +42,9 @@ data:
 
   # If you want to run cilium in debug mode change this value to true
   debug: "false"
+  # Set the TCP port for the agent health status API. This is not the port used
+  # for cilium-health.
+  agent-health-port: "9876"
```

Comment:
I see we're in good company on this port ;)

https://www.speedguide.net/port.php?port=9876

On a more serious note, if this is purely used for loopback traffic then it's probably not a terrible choice since it's more likely administrators have blacklisted it on the actual network due to some of the other famous users.

Reply:
Yeah, it's used for localhost-only traffic so we should be fine.